thruster/examples/jwt_example/jwt_context.rs

```diff
@@ -1,18 +1,15 @@
-#[macro_use]
-extern crate serde_json;
-extern crate frank_jwt;
-
-use snafu::{ResultExt, Snafu};
-
-use log::info;
-use serde_json::{json, Value};
-use thruster::core::response::Response;
-use thruster::errors::ThrusterError;
-use thruster::{async_middleware, middleware_fn};
-use thruster::{map_try, App, Request, Server, ThrusterServer};
-use thruster::{MiddlewareNext, MiddlewareResult};
-
-struct JwtContext {
+use bytes::Bytes;
+use std::collections::HashMap;
+use std::str;
+use thruster::{
+    middleware::{
+        cookies::{Cookie, CookieOptions, HasCookies, SameSite},
+        query_params::HasQueryParams
+    },
+    Context, Request, Response,
+};
+
+pub struct JwtContext {
```

Comment:
If you want to use a custom context, you must implement the `Context` trait, you can find that [here](https://github.com/trezm/Thruster/blob/master/thruster/src/core/context.rs).

Reply:
Wait, but I am implementing `Context` though. Check out line number 134. I'm implementing `Context` for `JwtContext`

Reply:
My fault, for some reason the diff I was looking at didn't have passed line 21!

Reply:
No worries. Don't merge this yet. I'm noticing some issue on my side. I'll let you know once it's fixedx